geniza/corpus/templates/corpus/document_transcribe.html

```diff
@@ -20,6 +20,7 @@
 {% endblock extrastyle %}
 {% block extrascript %}
     {% render_bundle_csp "annotation" "js" attrs='defer' %}
+    <script src="https://cdn.tiny.cloud/1/dxa2pi2u1fimtzb8r4t1mtpfckgiuj5vbpk3tdmxyxwdtu67/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

Comment:
This is fine as long as we document it clearly in tahqiq setup instructions.  

However, given that this has a code embedded in the url, it should be configured in django settings. We should put the Tiny API key in django local settings, include it in a context processors (either template globals or maybe just the transcribe view, since we only need it on this page).

I can add it to the settings for production/qa in the ansible deploy scripts.

Reply:
Done, and I also added a line of CSS that will be useful for upcoming changes in block labeling, to avoid opening a new PR here just for that!